spec/integrations/compatibility_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "bg_helper"
+require "anycable/rails/compatibility"
+
+describe "Compatibility" do
+  describe "Channel", :rpc_command do
+    include_context "rpc stub"
+
+    let(:data) { { action: 'follow' } }
+    let(:channel) { 'TestChannel' }
+
+    subject { service.command(request) }
+
+    describe "Channel#stream_from" do
+      let(:command) { 'message' }
+
+      it "returns ERROR when callback is passed" do
+        allow_any_instance_of(TestChannel).to receive(:follow) { |channel| channel.stream_from("all", -> {}) }
+        expect(subject.status).to eq :ERROR
+        expect(subject.error_msg).to include "Custom stream callbacks are not supported in AnyCable!"
+      end
+
+      it "returns ERROR when coder is passed" do
+        allow_any_instance_of(TestChannel).to receive(:follow) { |channel| channel.stream_from("all", coder: ActiveSupport::JSON) }
+        expect(subject.status).to eq :ERROR
+        expect(subject.error_msg).to include "Custom stream callbacks are not supported in AnyCable!"
```

Comment:
I think, we should modify the message, smth like: "Custom coders are not supported".

Also, we can support `json` coder; it's the implicit Action Cable coder, so we don't have to raise an exception

Reply:
Done, I guess in this case we want to use that valid coder somehow - right now our channel looks this way:

```ruby
module Channel
  class Base
    # ...

    def stream_from(broadcasting, _callback = nil, _options = {})
      connection.socket.subscribe identifier, broadcasting
    end

    # ...
  end
end
```